packages/blocks/src/rating-scale-answer/button.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import { RawHTML } from '@wordpress/element';
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import { Button, FormCheckbox } from '../components';
+
+const ButtonContent = styled.span`
```

Comment:
As this is basically copied over from `multiple-choice-answer`, we should think about making this into its own thing maybe.
But could be considered outside of the scope for this PR.